environments/hca-prod/terraform/buckets.tf

```diff
@@ -234,7 +247,14 @@ resource google_service_account_iam_binding hca_workload_identity_binding {
   service_account_id = module.hca_argo_runner_account.id
   role               = "roles/iam.workloadIdentityUser"
   members            = ["serviceAccount:${local.prod_project_id}.svc.id.goog[hca/argo-runner]"]
-  depends_on         = [module.hca_argo_runner_account]
```

Comment:
why is this line removed? if the argo runner account still exists, we do need it to be created before the binding, so this depends_on should stay

Reply:
the `depends_on` isn't present in the dev environment, which makes me think this line is redundant. you don't need to explicitly mark dependencies in terraform unless the dependent module's attributes aren't referenced in the resource (or if there's a bug in terraform's dependency resolution)

Reply:
more info in the docs here: https://www.terraform.io/docs/language/meta-arguments/depends_on.html

Reply:
TIL! Thanks for that